src/components/GamesList/index.tsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import "./GamesList.css";
+import UserInfo from "../UserInfo"
+import { Button, Container, Segment, SegmentGroup, Grid, GridRow, GridColumn } from "semantic-ui-react";
+
+export interface GameListEntry {
+	gameID: number,
+	playerNames: string[]
+}
+
+interface GamesListProps {
+    games: GameListEntry[];
+    onJoin: () => void;
+}
+
+interface ListEntryProps {
+    game: GameListEntry;
+    onJoin: () => void;
+}
+
+interface TopBarProps {
+    gameID: number;
+    onJoin: () => void;
+    full: boolean;
+}
+
+const MAX_PLAYERS = 4;
+const GAME_NAME = "Hearts";
+
+const GamesList: React.FC<GamesListProps> = ( {games, onJoin} ) => {
+    return (<SegmentGroup> {games.map((game) => {
+        return <ListEntry game={game} onJoin={onJoin}/>
+    })}</SegmentGroup>)
+};
+
+const ListEntry: React.FC<ListEntryProps> = ( {game, onJoin} ) => {
+    const emptySlotName = "Slot Empty"
+    let full = true;
+    const playerNames = game.playerNames;
+    if (playerNames.includes(emptySlotName)) {
+        full = false;
+    }
+    if (MAX_PLAYERS > playerNames.length) {
+        full = false;
+        const emptySpots = MAX_PLAYERS - playerNames.length
+        for (let i = 0; i < emptySpots; i++) {
+            playerNames.push(emptySlotName)
+        }
+    }
+
+    const playerAvatars = [
+        "https://upload.wikimedia.org/wikipedia/commons/thumb/3/3c/A_Shiba_Inu.jpg/64px-A_Shiba_Inu.jpg",
+        "https://upload.wikimedia.org/wikipedia/commons/thumb/6/6d/2._Mestre_%28Italy%29%2C_the_dog_MILO.jpg/64px-2._Mestre_%28Italy%29%2C_the_dog_MILO.jpg",
+        "https://upload.wikimedia.org/wikipedia/commons/thumb/3/31/Dalmapointer.jpg/64px-Dalmapointer.jpg",
+        "https://upload.wikimedia.org/wikipedia/commons/thumb/6/68/A_Shiba_Inu_enjoying_a_sunny_day.jpg/32px-A_Shiba_Inu_enjoying_a_sunny_day.jpg",
+    ]
+
+    const unknownAvatar = "https://upload.wikimedia.org/wikipedia/commons/thumb/5/55/Question_Mark.svg/64px-Question_Mark.svg.png";
+
+    let count = 0;
+
+    return (
+        <SegmentGroup>
+            <TopBar gameID={game.gameID} onJoin={onJoin} full={full}/>
+            <Grid columns={5} padded>
+                <GridRow className="game-list-row">   
+                    <GridColumn>
+                        <Container textAlign="center">
```

Comment:
I see what you mean about the container, I think they are being confusing and using this for two things: as a "text container" with either the `text` flag or a property like `text align`, like you have here **or** as a class container without any text property or flag set like the link I attached to the storybook.

The CSS documentation in the link I sent is much clearer about its normal use, the React documentation is confusing and, I agree, really only emphasizes it's use as a text container. One thing I've noticed about some of these React versions of other frameworks is that they are really focused on telling you about the React stuff, but they leave some of the information about how things are supposed to be used to the regular CSS documentation.

Reply:
Yeah I see now. The documentation for semantic ui is also clearer than the documentation for the react version.